packages/reports/reports.hjson

```diff
@@ -0,0 +1,212 @@
+    "id": "8",
+    "config": {
+      "fetch": [
+        {
+          "fetchAnalytics": {
+            "dataElementCodes": [
+              "BCD46",
+              "BCD47",
+              "BCD48",
+              "BCD48a",
+              "BCD48b",
+              "BCD49",
+              "BCD50",
+              "BCD51",
+              "BCD52",
+              "BCD53",
+              "BCD54",
+              "BCD55"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": true
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+          "transform": "select",
+          "$.dataElement": "$.value",
+          "...": ["period", "organisationUnit"]
+        },
+        {
+          "transform": "sort",
+          "by": "$.period",
+        },
+        {
+          "transform": "aggregate",
+          "organisationUnit": "group", 
+          "...": "last"
+        },
+        {
+          "transform": "aggregate",
+          "organisationUnit": "drop",
+          "period": "drop",
+          "...": "sum"
+        },
+        {
+          "transform": "select",
+          "'Doctors'": "$.BCD46",
+          "'Midwives'": "$.BCD47",
+          "'Nurses'": "add([$.BCD48, $.BCD48a, $.BCD48b])",
+          "'Aides'": "$.BCD49",
+          "'Others'": "add([$.BCD50, $.BCD51, $.BCD52, $.BCD53, $.BCD54, $.BCD55])"
+        }
+      ],
+      "output": [
+          {
+            "Doctors": 2,
+            "Midwives": 1,
+            "Nurses": 12,
+            "Aides": 7
+          }
+      ]
+    }
+  },
+  {
+    "id": "TO_RH_Descriptive_FP01_02",
+    "config": {
+      "fetch": [
+        {
+          "fetchAnalytics": {
+            "dataElementCodes": [
+              "Family_Planning_Acceptors_Other",
+              "Family_Planning_Acceptors_Condom_Female",
+              "Family_Planning_Acceptors_Natural_Method",
+              "Family_Planning_Acceptors_Depo_Provera",
+              "Family_Planning_Acceptors_Jadelle",
+              "Family_Planning_Acceptors_Pill_Single",
+              "Family_Planning_Acceptors_Sterilization_Male",
+              "Family_Planning_Acceptors_Sterilization_Female",
+              "Family_Planning_Acceptors_Pill_Combined",
+              "Family_Planning_Acceptors_Condom_Male",
+              "Family_Planning_Acceptors_IUD",
+              "POP21AND23",
+              "POP15",
+              "POP17AND19",
+              "POP13",
+              "POP11"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": false
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+            "transform": "select",
+            "$.dataElement": "$.value",
+            "'period'": "$.period"
+        },
+        {
+          "transform": "aggregate",
+          "period": "group",
```

Comment:
```suggestion
          "period": "combine",
```
I feel this could be easier for data stewards to understand. What do you think? Perhaps `group` is more related to the SQL groupBy?

Reply:
Another option `match`

Reply:
I think I slightly prefer `combine` to `match`. Honestly I've struggled over the right term for this one for a bit, nothing that I've come up with quite captures the right idea. Yeah I chose `group` intially as I was thinking about SQL `groupBy` at the time, but I don't think it's the best option...

I've also considered something like this:
```
{
    "transform": "aggregate",
    "per": ["period", "organisationUnit"],
    "BCD1": "sum",
    "...": "count"
}
``` 

Our current aggregation names all follow a convention of `<aggregationType>Per[ period | organisationUnit ]` eg. `sumPerPeriod`. I think it's a good way of describing the operation but I don't know the best way to re-use that naming convention through config here...